templates/templates/_navigation-developer-h.html

```diff
@@ -1,82 +1,230 @@
 <div class="fade-animation u-hide" id="developer-tab-content">
   <div class="dropdown-window__content">
-    <div class="dropdown-window__summary-strip">
-      <div class="row">
-        <p class="dropdown-window__summary">Dive into the docs for <a href="https://help.ubuntu.com/" title="Visit Ubuntu help - external site">Ubuntu desktop and server</a>, <a href="https://docs.ubuntu.com/" title="Visit the Ubuntu Docs - external site">MAAS, Juju, Snapcraft and Ubuntu Core</a>, or use our step-by-step <a href="https://tutorials.ubuntu.com/" title="Visit the Ubuntu Tutorials - external site">Ubuntu Tutorials</a> to try anything from installing Ubuntu to deploying OpenStack and Kubernetes.</p>
-        <hr>
+    <div class="p-strip is-x-shallow u-hide--small u-no-padding--bottom">
+      <div class="row u-equal-height">
+        <div class="tablet-col-2 col-3 p-card--navigation">
+          <h4 class="is-dense">
+            <a href="/desktop/developers">Development workflow&nbsp;&rsaquo;</a>
```

Comment:
I don't think we put `>` on title links

Reply:
As discussed this is to keep consistency with enterprise menu